### DISCUSSION TITLE
Rails 7.1 Support

### DISCUSSION THREAD

**Discussion by chadwilken** [mongodb/mongoid#5731] [General]
Looking for information about when we can expect a release adding Rails 7.1 support? I saw there was a merged PR but haven't seen a release or anything since then. This is the last remaining gem before we can update.

**Comment by jamis** [comment]
As you noted, there's work in progress. We are hopeful that a new release will be "soon", but we can't commit to anything more concrete than that at the moment. Stay tuned!

**Reply by Stf-F** [reply]
Hi @jamis, as a FYI the [docs](https://www.mongodb.com/docs/mongoid/current/installation/) already refer to v9.0 as if it was already [available](https://www.mongodb.com/docs/mongoid/master/release-notes/mongoid-9.0/).
The entire startup guide is rather misleading at best.
A little bit puzzling, and frustrating, especially for newcomers getting to grips with the environment. 
There is nothing personal in the above of course. 
Will wait on your update!
Cheers,

<img width="852" alt="image" src="https://github.com/mongodb/mongoid/assets/52630104/671c5dbc-ee61-4100-b889-da4fd26aaa5f">

<img width="790" alt="image" src="https://github.com/mongodb/mongoid/assets/52630104/fefeff69-885b-4722-9e0c-ddf3ab873a91">

**Reply by jamis** [reply]
Oh, no! That was certainly not intentional. I'm looking at that right now to see about rolling that change back. 9.0.0 is definitely not yet released--but hopefully will be in the not-too-distant future. Thanks for bringing this to our attention!

**Reply by jamis** [reply]
Thanks again for pointing out the problem with the documentation. That should all be resolved now, with the documentation now referring once again to the 8.1-stable branch.

**Reply by Stf-F** [reply]
No problem. The setup guide is fixed but the [release notes](https://www.mongodb.com/docs/mongoid/master/release-notes/mongoid-9.0/) still describe the version 9.0 though. ;)

**Reply by jamis** [reply]
Ah, that's intentional -- you're looking at the docs for "master" there (see the URL). Make sure you're looking at the "current" docs, and you'll be golden. Unless, of course, you reached the "master" release notes by clicking a link in the "current" docs? *That* would be a bug!

**Reply by jamis** [reply]
To be clearer: if you go to https://www.mongodb.com/docs/mongoid (or https://www.mongodb.com/docs/mongoid/current) you'll get the docs for the current stable release. If you see any mention of 9.0 in those docs, that's a bug. Sorry for the confusion!

**Reply by Stf-F** [reply]
Ok! Thanks for taking the time to clarify @jamis .

**Comment by comandeo** [comment]
Mongoid versions 8.0.7 and 8.1.3 that support Rails 7.1 were released last week.

**Reply by icortex** [reply]
@comandeo Are you planning to support Rails 7.1 in Mongid 7.x?

**Reply by comandeo** [reply]
Hi @icortex , we only backport critical bugfixes to Mongoid 7.x. May I ask what is keeping you from using Mongoid 8?
